vllm/model_executor/layers/fused_moe/cutlass_moe.py

```diff
@@ -178,6 +175,8 @@ def run_cutlass_moe_fp8(
         c2 = _resize_cache(workspace2, (M * topk, N))
         c3 = _resize_cache(workspace13, (M * topk, K))
 
+    c1.fill_(0)
```

Comment:
Can we have a condition here that we only zero-out `c1` if `expert_map` is not none and `per_act_token == True`? As far as I'm aware, this is the only case when it's needed

Reply:
There's another PR that has the proper condition for this.  I don't want to have to rerun everything at this point.  I'll let that other PR push the better fix.